Day2ConsoleAppHomework/Program.cs

```diff
@@ -1,15 +1,44 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Day2ConsoleAppHomework
 {
+    enum TransactionType { Buy, Sell }
+
     class Program
     {
+        string currency = "DKK";
+
         static void Main(string[] args)
         {
+            string currency = "DKK";
```

Comment:
why do we need to have the second currency variable declaration ?

Reply:
Ups - forgot to remove it after I found the proper place for it. It will be removed.